dts/arm/st/c0/stm32c011X6.dtsi

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2024 Kickmaker
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <mem.h>
+#include <st/c0/stm32c0.dtsi>
```

Comment:
There should be another `stm32c011.dtsi` file where the soc compatible is declared, similar to `stm32c031.dtsi`.
It's what should be included here.

Reply:
Or I think you could just include the `stm32c031.dtsi` instead, since the c011 & c031 share the same memory organization & GPIOs https://www.st.com/resource/en/reference_manual/rm0490-stm32c0x1-advanced-armbased-32bit-mcus-stmicroelectronics.pdf

Reply:
@RomainPelletant You would have to add `gpiod` as well, so I think just including `stm32c031.dtsi` instead would be the right way to do it.

Reply:
Thank you for explanation. Changes pushed